app/User.php

```diff
@@ -1,24 +1,19 @@
 <?php
-
```

Comment:
Again, please refrain from removing these spaces because these spaces are making the code readable.

Reply:
I've fixed it back.